### PR TITLE
Sylvia/docs/anant calculator

### PR DESCRIPTION
**Pull request by srpearce**
@mhoeber @lamagnifica @catong @JAAkana This PR adds information about the edX calculator (including screen shots), per a request from Anant. Please let me know what you think. Thanks!

### CODE REVIEW

File: en_us/course_authors/source/students/math_students.rst [hunk 1]
```diff
@@ -4,141 +4,8 @@
...
-
-- Trig functions: sin, cos, tan, sec, csc, cot
-- Their inverses: arcsin, arccos, arctan, arcsec, arccsc, arccot
-- Other common functions: sqrt, log10, log2, ln, exp, abs
-- Factorial: ``fact(3)`` or ``factorial(3)`` are valid. However, you must take
-  care to only input integers. For example, ``fact(1.5)`` would fail.
-- Hyperbolic trig functions and their inverses: sinh, cosh, tanh, sech, csch,
-  coth, arcsinh, arccosh, arctanh, arcsech, arccsch, arccoth
\ No newline at end of file
+This topic has moved! See `Entering Mathematical and Scientific Expressions
```
Comment (lamagnifica) [inline en_us/course_authors/source/students/math_students.rst:7]: this is great!

File: en_us/students/source/SFD_mathformatting.rst [hunk 2]
```diff
@@ -22,28 +25,60 @@ For some math, science, and other problems, you'll enter a numerical or math exp
...
+.. image:: /Images/Calc_Showing.png
+  :width: 600
+  :alt: Courseware page with the calculator visible along the edge of the browser
+
+Both the calculator and the response fields in math problems accept a selection
+of characters that represent numbers, operators, constants, functions, and other
+mathematical concepts. You may recognize parts of this system if you've used
+math programs before.
+
+.. note:: You can select the circled ``i`` icon on the right side of the calculator input field to see a pop-up hint that has an abbreviated list of these characters.
```
Comment (lamagnifica) [inline en_us/students/source/SFD_mathformatting.rst]: pro tip: you _can_ wrap a note! just put a hard carriage return at the appropriate point in the first line, and then indent the second line 1-2 spaces. When you do ctrl+alt+q on the 2nd line, the rest of the note text will wrap correctly.

File: en_us/students/source/SFD_mathformatting.rst [hunk 3]
```diff
@@ -22,28 +25,60 @@ For some math, science, and other problems, you'll enter a numerical or math exp
...
+  lowercase letters carefully.
 
 For more information about the types of characters you can use, see below.
 
 .. note:: The edX system accepts both constants and metric affixes. Be careful to distinguish between constants and metric affixes. Constants stand alone, while metric affixes must be combined with numbers.
 
-  For example, ``c`` can be a constant representing the speed of light or a metric affix meaning "centi". When you use ``c`` as a metric affix, join ``c`` with the number. When you use ``c`` as a constant, indicate multiplication explicitly. The following examples show the difference:
+  For example, ``c`` can be a constant representing the speed of light or a
+  metric affix meaning "centi". When you use ``c`` as a metric affix, join ``c``
+  with the number. When you use ``c`` as a constant, indicate multiplication
```
Comment (lamagnifica) [inline en_us/students/source/SFD_mathformatting.rst]: perhaps say what join means? (without spaces)

**Comment by lamagnifica** [comment]
:+1:

File: en_us/students/source/SFD_mathformatting.rst (hunk 2 above)
Comment (srpearce) [inline en_us/students/source/SFD_mathformatting.rst]: Oh, cool! Will do that.

**Comment by srpearce** [comment]
@catong "Clicking" works for computers, but not for mobile - I was trying to go with something platform-agnostic (I almost said "platform-androgynous" just now, but that's not quite right).

**Pull request merged**